boards/arm/thingy91_nrf9160/Kconfig

```
@@ -4,6 +4,19 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
+if BOARD_THINGY91_NRF9160 || BOARD_THINGY91_NRF9160_NS
+
+config BOARD_INIT_PRIORITY
```

```suggestion
config BOARD_CONFIG_INIT_PRIORITY
```
? 

Perhaps this reflects the purpose better?